crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -803,15 +825,21 @@ pub enum ConvertFlakeRefError {
 
     #[error("Unsupported git remote URL: {0}")]
     UnsupportedGitUrl(url::Url),
+
+    #[error("Failed to parse URL")]
+    URLParseFailed(#[from] crate::nix::url_parser::UrlParseError),
```

Comment:
can we import UrlParseError as `use runix::url_parser_UrlParseError`, its kind of weird being the only absolute path in the enum